core/src/main/bin/questdb.sh

```diff
@@ -201,10 +201,10 @@ function start {
     if [ "${QDB_CONTAINER_MODE}" != "" ]; then
         ${JAVA} ${JAVA_OPTS} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt
     elif [ "${QDB_DISABLE_HUP_HANDLER}" = "" ]; then
-        ${JAVA} ${JAVA_OPTS} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt &
+        ${JAVA} ${JAVA_OPTS} ${JVM_PREPEND} -p ${JAVA_LIB} -m ${JAVA_MAIN} -d ${QDB_ROOT} ${QDB_OVERWRITE_PUBLIC} > ${QDB_LOG}/stdout-${DATE}.txt &
```

Comment:
It'd look nicer if `JVM_PREPEND` would be added to `JAVA_OPTS` somewhere on lines 179-186. This way we won't need to add `JVM_PREPEND` to each start command.

Reply:
Thanks for the suggestion. I've update the PR